app/assets/javascripts/dashboard/pusher/pusher.js

```diff
@@ -0,0 +1,33 @@
+$(function() {
+  var pusherKey = $("#user").data("pusher-key");
+  var pusher = new Pusher(pusherKey);
+  var env = $("#user").data("environment");
+  var userId = $("#user").data("user-id");
+  var channel = pusher.subscribe("bg-job-" + env + "-notifier-" + userId);
+
+  notify = function(data) {
+    if (data.type == "success") {
+      var type = "success";
+      var message = data.message + " from <i>" +
+                    data.url + "</i>. Check them " +
+                    "<strong><a href='/blocks/" +
+                    data.block_id + "'>here</a></strong>";
+    } else{
+      var type = "danger";
+      var message = "Sorry, we got error for cards adding task: " +
+                    data.message;
+    };
+    var html = '<div class="col-sm-12"><div class="alert alert-dismissible' +
+               ' alert-' + type + '" ' + 'role="alert">' +
+               '<button type="button" class="close" data-dismiss="alert"' +
+               ' aria-label="Close"><span aria-hidden="true">&times;</span>' +
+               '</button><p id="alert">' + message + '</p></div></div>'
```

Comment:
Mixed double and single quotes.<br>'message' used out of scope.<br>Missing semicolon.